Okta/OktaAuth.swift

```diff
@@ -60,6 +62,46 @@ public struct OktaAuthorization {
     func passwordFlow(_ config: [String: String], credentials: [String: String]?, _ view: UIViewController) -> Promise<OktaTokenManager> {
         return buildAndPerformTokenRequest(config, additionalParams: credentials)
     }
+    
+    func logout(_ config: [String: String], idToken: String?, view: UIViewController) -> Promise<Void> {
```

Comment:
Since an `idToken` is required for this flow, lets make this a **required** function  argument instead of *optional*.